application.bs

```diff
@@ -0,0 +1,2350 @@
+<pre class='metadata'>
+Title: Open Screen Application Protocol
+Shortname: openscreenprotocol-application
+Level: None
+Status: w3c/ED
+ED: https://w3c.github.io/openscreenprotocol/application.html
+TR: https://www.w3.org/TR/openscreenprotocol/application.html
```

Comment:
We won't immediately publish the spec to /TR. And when we do, it will use the `openscreenprotocol-application` shortname, and thus be published at a different URL.

I think the line can be dropped for now.

```suggestion
```